code/game/objects/items.dm

```diff
@@ -461,6 +461,11 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 //Simple form ideal for basic use. That proc will return TRUE only when everything was done right, and FALSE if something went wrong, ot user was unlucky.
 //Editionaly, handle_failure proc will be called for a critical failure roll.
 /obj/item/proc/use_tool(var/mob/living/user, var/atom/target, base_time, required_quality, fail_chance, required_stat = null, instant_finish_tier = 110, forced_sound = null)
+	if (required_quality == QUALITY_WELDING)
```

Comment:
use your new variable here instead of qualities.

For example, a laser scalpel or an electric drill mitght also ignite